UPGRADING.rst

```diff
@@ -10,6 +10,23 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#4011 Integrate monitoring SNS topic with Slack
+===============================================
+
+Operator
+~~~~~~~~
+
+Set up AWS Chatbot Slack client, `described
+<./README.md#313-aws-chatbot-integration-with-slack>`_ in our README. This has
+been done for the `platform-hca-dev` account and remains to be configured for
+`platform-hca-anvildev` and `platform-hca-prod`.
+
+Follow these steps to deploy for ``dev.shared``, ``anvildev.shared``, and
```

Comment:
```suggestion
Upon completion of the above, follow these steps to deploy for ``dev.shared``, ``anvildev.shared``, and
```

Reply:
What about `prod`? I don't think you need to mention each deployment. Make sure checklist items exist for these steps.